### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by andreeaflorescu**
Changes are in the CHANGELOG.md file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 - Support for vm `KVM_SET_CLOCK`
 - Support for vm `KVM_GET_PIT2`
 - Support for vm `KVM_SET_PIT2`
+- Support for `KVM_GET_ONE_REG`
```
Comment (sameo) [inline CHANGELOG.md]: Don't you need to `s/Unreleased/v0.3.0/` as well?

Comment (andreeaflorescu) [reply CHANGELOG.md]: True that. Will do tomorrow.

Comment (andreeaflorescu) [reply CHANGELOG.md]: Done.

Comment (acatangiu) [reply CHANGELOG.md]: consistency nit:
```suggestion
- Support for vcpu `KVM_GET_ONE_REG`
```

**Review by acatangiu — DISMISSED** [review]

**Review by jiangliu — APPROVED** [review]

**Review by sameo — APPROVED** [review]

**Pull request merged**